src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+
+  if (sourceString.length === 0) {
+    return styles;
+  }
+
+  sourceString
+    .split(';')
+    .map((el) => el.trim())
+    .filter((el) => el.includes(':'))
+    .forEach((el) => {
+      const [key, ...values] = el.split(':');
+      const value = values.join(':').trim();
```

Comment:
The use of `...values` and `values.join(':')` is correct for handling cases where the value itself might contain colons. However, ensure that the `key` is properly trimmed before using it to avoid any leading or trailing spaces in the object keys.